dynamic/ec2readonly_policy.tf

```diff
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows read access to some EC2 attributes
+# in the dynamic account.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "ec2readonly_doc" {
+  statement {
+    actions = [
+      "ec2:DescribeAddresses",
+      "ec2:DescribeInstances",
+      "ec2:DescribeRegions",
+      "ec2:DescribeTags",
+    ]
+
+    resources = ["*"]
+  }
+}
```

Comment:
Thanks for mentioning the pre-built policy, but you nailed my thought process- I figure we should just roll our own policy in the name of "least privilege".  If we find other uses for this role that require more access, then it may make sense to use the pre-built policy.